There are a bunch of `maybe-uninitlaized` warnings when passing `-fexceptions` to the build.

`-fexceptions` is in the default CFLAGS that fedora wants to pass to all builds.
```
In file included from terminal.h:4:0,
                 from cmd.h:6,
                 from lpass.c:8:
lpass.c: In function ‘main’:
util.h:40:2: warning: ‘alias_val’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  free(*(void**) p);
  ^
lpass.c:86:23: note: ‘alias_val’ was declared here
  _cleanup_free_ char *alias_val;
                       ^
In file included from agent.c:9:0:
agent.c: In function ‘agent_kill’:
util.h:40:2: warning: ‘path’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  free(*(void**) p);
  ^
agent.c:173:23: note: ‘path’ was declared here
  _cleanup_free_ char *path;
                       ^
In file included from agent.c:9:0:
agent.c: In function ‘agent_start’:
util.h:40:2: warning: ‘path’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  free(*(void**) p);
  ^
agent.c:106:23: note: ‘path’ was declared here
  _cleanup_free_ char *path;
                       ^
In file included from agent.c:9:0:
agent.c: In function ‘agent_get_decryption_key’:
util.h:40:2: warning: ‘path’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  free(*(void**) p);
  ^
agent.c:208:23: note: ‘path’ was declared here
  _cleanup_free_ char *path;
                       ^
In file included from blob.c:9:0:
blob.c: In function ‘blob_write’:
util.h:40:2: warning: ‘version’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  free(*(void**) p);
  ^
blob.c:584:23: note: ‘version’ was declared here
  _cleanup_free_ char *version;
                       ^
In file included from terminal.h:4:0,
                 from cmd.h:6,
                 from cmd-ls.c:4:
cmd-ls.c: In function ‘print_node’:
util.h:40:2: warning: ‘timestr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  free(*(void**) p);
  ^
cmd-ls.c:102:27: note: ‘timestr’ was declared here
      _cleanup_free_ char *timestr;
                           ^
In file included from terminal.h:4:0,
                 from cmd.h:6,
                 from cmd-ls.c:4:
cmd-ls.c: In function ‘cmd_ls’:
util.h:40:2: warning: ‘timestr’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  free(*(void**) p);
  ^
cmd-ls.c:217:26: note: ‘timestr’ was declared here
     _cleanup_free_ char *timestr;
                          ^
In file included from endpoints.c:9:0:
endpoints.c: In function ‘lastpass_get_blob’:
util.h:40:2: warning: ‘blob’ may be used uninitialized in this function [-Wmaybe-uninitialized]
  free(*(void**) p);
  ^
endpoints.c:36:23: note: ‘blob’ was declared here
  _cleanup_free_ char *blob;
                       ^
```

--- a/agent.c
+++ b/agent.c
@@ -103,7 +103,6 @@ static int agent_socket_get_cred(int fd, struct ucred *cred)
 
 static void agent_run(unsigned const char key[KDF_HASH_LEN])
 {
-	_cleanup_free_ char *path;
 	char *agent_timeout_str;
 	unsigned int agent_timeout;
 	struct sockaddr_un sa, listensa;
@@ -123,7 +122,7 @@ static void agent_run(unsigned const char key[KDF_HASH_LEN])
 	if (agent_timeout)
 		alarm(agent_timeout);
 
-	path = agent_socket_path();
+	_cleanup_free_ char *path = agent_socket_path();
 	if (strlen(path) >= sizeof(sa.sun_path))
 		die("Path too large for agent control socket.");
 
@@ -170,12 +169,11 @@ static void agent_run(unsigned const char key[KDF_HASH_LEN])
 
 void agent_kill(void)
 {
-	_cleanup_free_ char *path;
 	struct sockaddr_un sa;
 	struct ucred cred;
 	int fd;
 
-	path = agent_socket_path();
+	_cleanup_free_ char *path = agent_socket_path();
 	if (strlen(path) >= sizeof(sa.sun_path))
 		die("Path too large for agent control socket.");
 
@@ -205,12 +203,11 @@ out:
 
 static bool agent_ask(unsigned char key[KDF_HASH_LEN])
 {
-	_cleanup_free_ char *path;
 	struct sockaddr_un sa;
 	int fd;
 	bool ret = false;
 
-	path = agent_socket_path();
+	_cleanup_free_ char *path = agent_socket_path();
 	if (strlen(path) >= sizeof(sa.sun_path))
 		die("Path too large for agent control socket.");
 

--- a/blob.c
+++ b/blob.c
@@ -581,12 +581,11 @@ size_t blob_write(const struct blob *blob, const unsigned char key[KDF_HASH_LEN]
 {
 	struct buffer buffer;
 	struct share *last_share = NULL;
-	_cleanup_free_ char *version;
 	struct account *account;
 
 	memset(&buffer, 0, sizeof(buffer));
 
-	version = xultostr(blob->version);
+	_cleanup_free_ char *version = xultostr(blob->version);
 	buffer_append(&buffer, "LPAV", 4);
 	write_plain_string(&buffer, version);
 	buffer_append(&buffer, "LOCL", 4);

--- a/cmd-ls.c
+++ b/cmd-ls.c
@@ -99,11 +99,9 @@ static void print_node(struct node *head, int level)
 				printf("    ");
 			if (node->account) {
 				if (long_listing) {
-					_cleanup_free_ char *timestr;
-					if (show_mtime)
-						timestr = format_timestamp(node->account->last_modified_gmt, true);
-					else
-						timestr = format_timestamp(node->account->last_touch, false);
+					_cleanup_free_ char *timestr = show_mtime ?
+						format_timestamp(node->account->last_modified_gmt, true) :
+						format_timestamp(node->account->last_touch, false);
 					terminal_printf(TERMINAL_FG_CYAN "%s ", timestr);
 				}
 				terminal_printf(TERMINAL_FG_GREEN TERMINAL_BOLD "%s" TERMINAL_NO_BOLD " [id: %s]" TERMINAL_RESET "\n", node->name, node->account->id);
@@ -214,11 +212,9 @@ int cmd_ls(int argc, char **argv)
 			insert_node(root, fullname, account);
 		else {
 			if (long_listing) {
-				_cleanup_free_ char *timestr;
-				if (show_mtime)
-					timestr = format_timestamp(account->last_modified_gmt, true);
-				else
-					timestr = format_timestamp(account->last_touch, false);
+				_cleanup_free_ char *timestr = show_mtime ?
+					format_timestamp(account->last_modified_gmt, true) :
+					format_timestamp(account->last_touch, false);
 				printf("%s ", timestr);
 			}
 			printf("%s [id: %s]\n", fullname, account->id);

--- a/endpoints.c
+++ b/endpoints.c
@@ -33,9 +33,8 @@ void lastpass_logout(const struct session *session)
 struct blob *lastpass_get_blob(const struct session *session, const unsigned char key[KDF_HASH_LEN])
 {
 	size_t len;
-	_cleanup_free_ char *blob;
 
-	blob = http_post_lastpass("getaccts.php", session->sessionid, &len, "mobile", "1", "requestsrc", "cli", "hasplugin", LASTPASS_CLI_VERSION, NULL);
+	_cleanup_free_ char *blob = http_post_lastpass("getaccts.php", session->sessionid, &len, "mobile", "1", "requestsrc", "cli", "hasplugin", LASTPASS_CLI_VERSION, NULL);
 	if (!blob || !len)
 		return NULL;
 	config_write_encrypted_buffer("blob", blob, len, key);

--- a/lpass.c
+++ b/lpass.c
@@ -83,11 +83,10 @@ static void expand_aliases(int *argc, char ***argv)
 	int argv_alloced;
 	int new_argc = 0;
 	_cleanup_free_ char *config_name;
-	_cleanup_free_ char *alias_val;
 
 	xasprintf(&config_name, "alias.%s", alias);
 
-	alias_val = config_read_string(config_name);
+	_cleanup_free_ char *alias_val = config_read_string(config_name);
 	if (!alias_val)
 		return;
 

